quipucords/scanner/network/runner/roles/redhat_release/tasks/main.yml

```diff
@@ -4,25 +4,17 @@
   set_fact:
     internal_host_started_processing_role: "redhat_release"
 
-- name: initialize redhat_release facts
```

Comment:
Won't that cause any impacts on the way the report is printed in case there's no - "'stdout_lines' in internal_redhat_release"?

Reply:
Mmm I think it will. Nice catch!